src/user/use-cases/list-user-by-id/list-user-by-id.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { NotFoundException } from '@nestjs/common';
+import { User } from '@/user/infra/entities';
+import { ListUserByIdUseCase } from './list-user-by-id';
+import { IUserRepository } from '@/user/infra/interfaces';
+
+describe('Get User', () => {
+  let listUserUseCase: ListUserByIdUseCase;
+  let userRepository: jest.Mocked<IUserRepository>;
+
+  const user_id = '123456';
+  const non_existent_user_id = 'non_existent_user_id';
+
+  beforeAll(() => {
+    userRepository = {
+      findById: jest.fn(),
+    } as jest.Mocked<IUserRepository>;
+
+    listUserUseCase = new ListUserByIdUseCase(userRepository);
+  });
+
+  it('should be able to return an user when found', async () => {
```

Comment:
Não precisa escrever uma descrição gigantesca.

Você pode ser mais objetivo e claro com uma descrição assim "should be able to return an user"

Reply:
Corretio, arrumei!